src/Requester/PsrClientRequester.php

```diff
@@ -15,18 +17,20 @@ class PsrClientRequester implements RequesterInterface
     private ClientInterface $client;
     private RequestFactoryInterface $requestFactory;
     private StreamFactoryInterface $streamFactory;
+    private ResponseFactoryInterface $responseFactory;
 
-    public function __construct(ClientInterface $client, RequestFactoryInterface $requestFactory, StreamFactoryInterface $streamFactory)
+    public function __construct(ClientInterface $client, RequestFactoryInterface $requestFactory, StreamFactoryInterface $streamFactory, ResponseFactoryInterface $responseFactory)
```

Comment:
I think that `responseFactory` should be optional. In case `null` (or nothing) is passed, the response factory could be created with the default implementation.
The same comment applies to `SymfonyHttpClientRequester` constructor also.

Reply:
Oky, done!